============================= test session starts ==============================
platform linux -- Python 3.8.0, pytest-5.2.1, py-1.8.0, pluggy-0.13.0
rootdir: /home/doctor/lab_2
collected 6 items

tests/tests.py ......                                                    [100%]

============================== 6 passed in 1.10s ===============================
========================================
Результат без параметрів: 
No URL passed to function
========================================
Результат з правильною URL: 
Time is:  10:41:16 AM
Date is:  10-23-2019
========================================
	 Добрий день!	
